lib/generators/curation_concerns/install_generator.rb

```diff
@@ -104,5 +105,12 @@ def resource_types_config
     def simple_form_bootstrap
       generate 'simple_form:install --bootstrap'
```

Comment:
It looks like that generator also generates some additional files. I assumed we wanted to let `simple_form` continue to generate these additional files:

- `config/locales/simple_form.en.yml`
- `lib/templates/erb/scaffold/_form.html.erb`

Would you rather we also generate those files even though it looks like we aren't overwriting them within CC?